HW7/index.js

```diff
@@ -0,0 +1,8 @@
+let firstNumber = prompt("First number");
+firstNumber = Number(firstNumber);
```

Comment:
It will be better to combine methods
```suggestion
let firstNumber = Number(prompt("First number"));
```